src/lib/journey/callbacks/terms-and-conditions/terms-conditions.story.svelte

```diff
@@ -1,10 +1,14 @@
 <script lang="ts">
   import type { TermsAndConditionsCallback } from '@forgerock/javascript-sdk';
+  import { initialize } from '$lib/links.store';
 
   import Centered from '$components/primitives/box/centered.svelte';
   import Terms from './terms-conditions.svelte';
 
   export let callback: TermsAndConditionsCallback;
+
+  // Initialize links
+  initialize({ termsAndConditions: 'https://www.forgerock.com/terms' });
```

Comment:
I chose the ForgeRock terms page as the default / test URL for Storybook, e2e and test apps. Open to suggestions on what the "default" within the project should be if there are any.